README.md

```diff
@@ -0,0 +1,155 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building [Capsule](https://github.com/capsule-rs/capsule) applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Getting Started](#getting-started)
+  * [Vagrant and Docker](#vagrant-and-docker)
+  * [Linux Distributions](#linux-distributions)
+  * [Without Docker](#without-docker)
+  * [Kernel NIC Interface](#kernel-nic-interface)
+* [Packaging for Release](#packaging-for-release)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Introduction
+
+`Capsule` is built on Intel's [Data Plane Development Kit](https://www.dpdk.org/) release 18.11. While it is written in Rust, it needs to be able to call `DPDK`'s C functions installed separately as shared dynamic libaries. To make developing and running Capsule applications easy, we created the sandbox container with all the necessary dependencies. You can pull down the container and start writing network functions in stable `Rust` right away.
+
+## Getting Started
+
+To run the [sandbox](https://hub.docker.com/repository/docker/getcapsule/sandbox), the docker host must run on a Linux distribution. `DPDK` requires either Linux or FreeBSD. We plan to add FreeBSD support in the future.
+
+### Vagrant and Docker
+
+The quickest way to start the sandbox is to use the VirtualBox virtual machine included in this repository. If you are developing on either MacOS or Windows, this is the only way to write a `Capsule` application.
+
+After cloning this repository, you can start the vagrant VM using commands,
+
+```
+host$ vagrant up
+host$ vagrant ssh
+```
+
+Now you are inside a `Debian` VM with `Docker` installed. The system is already preconfigured for DPDK. To run the sandbox, use the command,
+
+```
+vagrant$ docker run -it --rm \
```

Comment:
this would be nice to be a shell script and/or make target

Reply:
it is a make command, sorta. the make command we have assumes you are contributing to capsule. for this section, we want to target people also use capsule as a dependency for their own app. that's why it has the actual command and the details on how they should modify it to suit their needs.